content/posts/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: 'Posts'
+date: 2022-08-30T21:05:00+05:30
+draft: false
```

Comment:
Is this needed for anything?

Reply:
yeah .  to tell hugo this is a content root

Reply:
yeah to allow higo to pickup content.

Reply:
I understand why the file is needed, but `draft: ` and `date: `? `content/_index.md` doesn't have those fields either.